### PR TITLE
Russian translation for the app

### PR DESCRIPTION
**Pull request by captaincurls**
Hello! I made a Russian translation for this project, would appreciate it being added to the app :)

### CODE REVIEW

**Comment by captaincurls** [comment]
Updated the selector!

File: js/translations.js [hunk 1]
```diff
@@ -16,8 +16,11 @@ i18next
   const languages = [
     ['de', 'Deutsch'],
     ['en', 'English'],
+    ['fr', 'Français'],
```
Comment (mikebryant) [inline js/translations.js:19]: This and the Portuguese line shouldn't be changing in this PR - Not sure what happened here, I think you need to `git rebase` to remove conflicts

Comment (captaincurls) [reply js/translations.js:19]: Sorry for the confusion, it's my first time contributing. I will upload a file with only one line added shortly!

Comment (mikebryant) [reply js/translations.js]: No problem :)

**Pull request merged**